app/graphql/mutations/availabilities/create_availability.rb

```diff
@@ -7,11 +7,15 @@ class CreateAvailability < ::Mutations::BaseMutation
       def resolve(params:)
         availability_params = Hash params
         availability = Availability.create(availability_params)
+        schedule_pairing(availability) if availability
+        availability
+      end
+
+      def schedule_pairing(availability)
         open_slot = availability.find_availabilities_to_pair
         pairing = Pairing.create_pairing(availability, open_slot) unless open_slot.nil?
```

Comment:
This `.nil?` is really weird to me. I tried to do the same thing on update_availability but I had to do a `.empty?` check or else it failed. See line 22 of update availability. I'm just not sure why it would need to be different and it seems a bit fishy to me

Reply:
That is really weird. Let's chat IRL